stripe/src/main/java/com/stripe/android/model/CardFunding.kt

```diff
@@ -0,0 +1,14 @@
+package com.stripe.android.model
+
+enum class CardFunding(internal val code: String) {
+    Credit("credit"),
+    Debit("debit"),
+    Prepaid("prepaid"),
+    Unknown("unknown");
+
+    internal companion object {
+        internal fun fromCode(code: String?): CardFunding? {
+            return values().firstOrNull { it.code == code }
```

Comment:
Previously you checked with `ignoreCase=true`, do you wish to preserve that here too?

Reply:
That logic was unnecessary. The API always returns lowercase strings.